AssetEditor/Properties/PublishProfiles/FolderProfile.pubxml

```diff
@@ -1,13 +1,13 @@
-﻿<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <Configuration>Release</Configuration>
-    <Platform>Any CPU</Platform>
-    <PublishDir>bin\Release\netcoreapp3.1\publish\win-x64\</PublishDir>
-    <PublishProtocol>FileSystem</PublishProtocol>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-    <SelfContained>true</SelfContained>
-    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
-    <PublishSingleFile>true</PublishSingleFile>
-    <PublishReadyToRun>false</PublishReadyToRun>
-  </PropertyGroup>
+﻿<Project ToolsVersion="4.0">
+    <PropertyGroup>
+        <Configuration>Release</Configuration>
+        <Platform>Any CPU</Platform>
+        <PublishDir>bin\Release\publish\</PublishDir>
+        <PublishProtocol>FileSystem</PublishProtocol>
+        <TargetFramework>net6.0-windows</TargetFramework>
+        <SelfContained>false</SelfContained>
```

Comment:
should be set to true, so that .net dlls are baked in the program itself. 
for c++ DLL, we can include it as resource string later on and when the user run the program, the program will unpack it from its own internal resouce and loads them as if they were next to program itself (this part is low prio)

Reply:
Yes, I tried that, but I can try again, and did not work, or something, I can try again¨'

Reply:
ok then, we can leave it as false for now
We should come back again, and set this back to true. (high prio)
I will make issue about this to remind ourselves.